connaisseur/tests/integration/cosign-update.yaml

```diff
@@ -0,0 +1,22 @@
+deployment:
+  imagePullPolicy: Never
+notary:
+  host: notary.docker.io
+  selfsigned: false
+  auth:
+    enabled: false
+  rootPubKey: |
+    -----BEGIN PUBLIC KEY-----
+    MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEUU2/8oNKPRTVjnTPY4WwUNFXYke3
+    CmpJpSaqBOFh92bdXZENv0AdR5qipqWxjGWO28Miyc2O6vGuoIyzzhQvuA==
+    -----END PUBLIC KEY-----
+  isCosign: true
```

Comment:
Would mark the experimental in the name of the var e.g. `isCosign(experimental)` to be on the very safe side

Reply:
you meant in the `helm/values.yaml` right? There is a comment above.

Reply:
I know, that's why I said "very" :-)

Reply:
I made a comment in the helm/values.yaml just above the `isCosign`. The advantage of a comment would be that in case someone is using it, they will not have to change the flag when it becomes non-experimental, but I am open about it. Opinions?

Reply:
I'd say with the comment in README, COSIGN and values.yml and the fact that cosign itself is experimental it's fine to not have it in the name

Reply:
fine with me :-)